CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 
 ## vNEXT
-- 
 
-## v1.0.0
\ No newline at end of file
+- [BREAKING] Update: Transition to RLC as the Unit in Subgraph Fields. The unit for several fields in the subgraph will be updated from nRLC to RLC. The impacted fields include:
+  - appPrice
+  - datasetPrice
+  - workerpoolPrice
+  - sponsoredAmount
+  - drainedAmount
+  - refundedAmount
+  - topUpValue
+  - value
+
+- Add unit tests for Event:
+  - voucherCreated
```

Comment:
You can eventually add PR number like that https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/blob/main/CHANGELOG.md?plain=1#L9